x/sequencer/params_migrations.go

```diff
@@ -34,7 +34,27 @@ func NewMigrator(keeper *seqkeeper.Keeper, ss Subspace) Migrator {
 // Migrate2to3 migrates from version 2 to 3.
 func (m Migrator) Migrate2to3(ctx sdk.Context) error {
 	var currParams types.Params
-	m.legacySubspace.GetParamSet(ctx, &currParams)
+	m.legacySubspace.Get(ctx, types.KeyMinBond, &currParams.MinBond)
+	m.legacySubspace.Get(ctx, types.KeyKickThreshold, &currParams.KickThreshold)
+	m.legacySubspace.Get(ctx, types.KeyNoticePeriod, &currParams.NoticePeriod)
+	m.legacySubspace.Get(ctx, types.KeyLivenessSlashMinMultiplier, &currParams.LivenessSlashMinMultiplier)
+	m.legacySubspace.Get(ctx, types.KeyLivenessSlashMinAbsolute, &currParams.LivenessSlashMinAbsolute)
+
+	if currParams.MinBond.Denom == "" {
```

Comment:
this specifically we need to change to 100 (it's set now to 1000 on mainnet)

Reply:
They don't exist on mainnet, but I can't say for other networks

Reply:
yea I mean they don't exist from previous version we're upgrading from (mainnet, testnet etc) so I don't think its network dependent. either way..